Menu/Menu.js

```diff
@@ -8,8 +8,35 @@ let menuItems = [
   'Music',
   'Log Out'
 ];
+const header = document.querySelector('.header')
+const menuButton = document.querySelector('.menu-button')
 
-/* 
+function createMenuComponent(menuItems) {
+  const menu = document.createElement('div')
+  menu.classList.add('menu')
```

Comment:
I like the grouping of element creating and class adding here. It makes it easier to see everything that is happening to each element in one place.